src/test/kotlin/com/zepben/evolve/services/common/UnknownPrimitivesTest.kt

```diff
@@ -1,28 +0,0 @@
-/*
- * Copyright 2020 Zeppelin Bend Pty Ltd
- *
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at https://mozilla.org/MPL/2.0/.
- */
-package com.zepben.evolve.services.common
-
-import com.zepben.testutils.junit.SystemLogExtension
-import org.hamcrest.MatcherAssert.assertThat
-import org.hamcrest.Matchers.equalTo
-import org.junit.jupiter.api.Test
-import org.junit.jupiter.api.extension.RegisterExtension
-
-internal class UnknownPrimitivesTest {
-
-    @JvmField
-    @RegisterExtension
-    var systemErr: SystemLogExtension = SystemLogExtension.SYSTEM_ERR.captureLog().muteOnSuccess()
-
-    @Test
-    internal fun checkEqualsOperator() {
-        assertThat(UNKNOWN_DOUBLE == UNKNOWN_DOUBLE, equalTo(true))
-        assertThat(UNKNOWN_INT == UNKNOWN_INT, equalTo(true))
```

Comment:
Equals operator does not need to be checked here: for any `val x: Double` or `val x: Int`, `x == x` is guaranteed.

Reply:
I suspect this was for coverage at some stage and agree it is no longer relevant